clis/teliod/src/configure_interface.rs

```diff
@@ -24,9 +24,10 @@ fn execute(command: &mut Command) -> Result<(), TeliodError> {
     }
 }
 
-#[derive(Debug, Deserialize, PartialEq, Eq)]
+#[derive(Default, Debug, Deserialize, Serialize, PartialEq, Eq)]
 #[serde(rename_all = "lowercase")]
 pub enum InterfaceConfigurationProvider {
+    #[default]
     Manual,
     Ifconfig,
     Iproute,
```

Comment:
This is just an idea, so you may resolve this message once you have read it.

Eventually it would be really nice to use netlink for network configs instead of spawning shell commands :).

Reply:
I think is a good idea but out of scope of this PR :)